### DISCUSSION TITLE
SmoothGraphics's scaleMode property should be optional.

### DISCUSSION THREAD

**Discussion by humbleCoder1990** [pixijs/pixijs#7735] [Q&A]
![image](https://user-images.githubusercontent.com/87828192/130339403-6376a5e8-4f52-4ab5-a5a3-4558f32a18f0.png)

It's ok when I pass arguments to lineStyle()
But if I pass LineStyleOptions, TS give this warning.

**Comment by bigtimebuddy** [comment]
@humbleCoder1990 This discussion board isn't the right please for this. Please file issues related to smooth graphics here: https://github.com/pixijs/graphics-smooth/issues

**Reply by humbleCoder1990** [reply]
I'm sorry Matt.
I'll go to there.

**Comment by ivanpopelyshev** [comment]
oops! thank you!

**Reply by ivanpopelyshev** [reply]
wanna make a PR for that? its easy
